pkg/apis/messaging/v1/subscription_validation_test.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/google/go-cmp/cmp"
 	"knative.dev/pkg/apis"
 	duckv1 "knative.dev/pkg/apis/duck/v1"
+	"knative.dev/pkg/ptr"
```

Comment:
why are we here using different `Int32()` than the other versions?

Reply:
pointer.Int32 was added after, so it's not available.